verifier/util/fake_attestation_server.go

```diff
@@ -11,13 +12,17 @@ import (
 )
 
 const fakeAsHostEnv = "GOOGLE_APPLICATION_CREDENTIALS"
```

Comment:
Why do we need to set this? And why is it set in the attestation server fake?

Reply:
Application Default Credentials (ADC) is a strategy used by the Google authentication libraries to automatically find credentials based on the application environment. 
ADC searches for credentials in GOOGLE_APPLICATION_CREDENTIALS environment variable first (https://cloud.google.com/docs/authentication/application-default-credentials).
We use fakeAsHostEnv to let ADC find fake credential.
You are right, moving to fake_oauth2_server

Reply:
But what component is actually fetching ADC and what error is it getting?